packages/server/src/models/UserItemModel.ts

```diff
@@ -118,7 +118,7 @@ export default class UserItemModel extends BaseModel<UserItem> {
 	}
 
 	public async addMulti(userId: Uuid, itemsQuery: Knex.QueryBuilder | Item[], options: SaveOptions = {}): Promise<void> {
-		const items: Item[] = Array.isArray(itemsQuery) ? itemsQuery : await itemsQuery.whereNotIn('id', this.db('user_items').select('item_id').where('user_id', '=', userId));
+		const items: Item[] = Array.isArray(itemsQuery) ? itemsQuery : await itemsQuery?.whereNotIn('id', this.db('user_items').select('item_id').where('user_id', '=', userId));
```

Comment:
Dropping `?` here and there and silencing errors is not a fix. `itemsQuery` should not be null so the first step would to be understand why it was.